lib/rules/block-spacing.js

```diff
@@ -15,7 +15,8 @@ var util = require("../ast-utils");
 module.exports = function(context) {
     var always = (context.options[0] !== "never"),
         message = always ? "Requires a space" : "Unexpected space(s)",
-        sourceCode = context.getSourceCode();
+        sourceCode = context.getSourceCode(),
+        tokensAndComments = sourceCode.tokensAndComments;
```

Comment:
That's a good point, though we would have to index each comment, which we don't currently do. And then we would need methods to get the comments (`getTokenOrCommentBefore`?)